tests/ff-server-sdk-test/ConfigTest.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using io.harness.cfsdk.client.api;
+using io.harness.cfsdk.client.cache;
+using io.harness.cfsdk.client.dto;
+using io.harness.cfsdk.HarnessOpenAPIService;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+
+namespace ff_server_sdk_test
+{
+    public class ConfigTests
+    {
+        [Test]
+        public void DefaultConfigAnalyticsSetCorrectly()
```

Comment:
Can we add a test for testing that non-default values get written/set correctly as well?

Reply:
Will do!